tools/eeprom-tool/cam96tof1_eeprom.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * BSD 3-Clause License
+ *
+ * Copyright (c) 2020, Analog Devices, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from
+ *    this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+#ifndef CAM96TOF1_EEPROM_H
+#define CAM96TOF1_EEPROM_H
+
+#include <aditof/system.h>
+#include <aditof/eeprom_interface.h>
+#include "camera_eeprom_interface.h"
+
+#include <memory>
+
+
+class Camera96Tof1Eeprom: public CameraEepromInterface {
+    public:
+        Camera96Tof1Eeprom(std::shared_ptr<aditof::EepromInterface> _eeprom);
+        virtual aditof::Status read(std::vector<uint8_t>& data) override;
+        virtual aditof::Status write(const std::vector<uint8_t>& data) override;
+    private:
+        std::shared_ptr<aditof::EepromInterface> m_eeprom;
+};
+
+#endif // CAM96TOF1_EEPROM_H
```

Comment:
new lines are still missing at the eof. Are you using the clang-format?

Reply:
I was not. now it is enabled.